src/rootPages/Designer/ui_work_interface_list_newPage_blank.js

```diff
@@ -0,0 +1,274 @@
+/*
+ * ui_work_interface_list_newPage_blank
+ *
+ * Display the form for creating a new ABPage.
+ */
+
+export default function (AB) {
+   var L = function (...params) {
+      return AB.Multilingual.labelPlugin("ABDesigner", ...params);
+   };
+
+   class UI_Work_Interface_List_NewPage_Blank extends AB.ClassUI {
+      constructor() {
+         var base = "ui_work_interface_list_newPage_blank";
+         super({
+            component: base,
+
+            form: `${base}_blank`,
```

Comment:
you can just leave these blank:  `form: "",`  and the super class will take care of the values.